tests/plotting/test_plotting.py

```diff
@@ -946,6 +946,13 @@ def test_box_axes():
     plotter.show(before_close_callback=verify_cache_image)
 
 
+def test_box_axes_label_color():
+    plotter = pyvista.Plotter()
+    plotter.add_axes(box=True, box_args={'label_color': 'red'})
+    plotter.add_mesh(pyvista.Sphere())
+    plotter.show(before_close_callback=verify_cache_image)
```

Comment:
Since this is a test with a new cache image we have to add it to the git repo. If you want to do this you'll have to run pytest with the `--reset...` flag as described at https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#notes-regarding-image-regression-testing . Let me know if I should do this for you.

Reply:
And this test is failing with a `TypeError`:
```
        if interactive is None:
            interactive = self._theme.interactive
        if hasattr(self, 'axes_widget'):
            self.axes_widget.EnabledOff()
            self.Modified()
            del self.axes_widget
        if box is None:
            box = self._theme.axes.box
        if box:
            if box_args is None:
                box_args = {}
>           self.axes_actor = create_axes_orientation_box(
                label_color=color,
                line_width=line_width,
                x_color=x_color,
                y_color=y_color,
                z_color=z_color,
                xlabel=xlabel,
                ylabel=ylabel,
                zlabel=zlabel,
                labels_off=labels_off,
                **box_args,
            )
E           TypeError: pyvista.plotting.tools.create_axes_orientation_box() got multiple values for keyword argument 'label_color'
```

Reply:
Will check the tests tonight, was a bit in a hurry yesterday and pushed a bit too fast it seems.
If you find the time to add the new cache image, that would be nice. I think it's better that it is generated on a Linux system (as the tests will only compare on linux in CI) or does that not matter? Otherwise I do it myself tonight :)

Reply:
Good point, there's a remark in the contributing docs that linux is better. Although I still see subtle differences compared to cache, potantially due to having no discrete GPU in my laptop, but it should be good enough. I'll push it.

Reply:
I just realised that the new cache image is exactly for the new failing test :) I'll see if I can easily fix the test then. Then the only remaining item for you would be the property renaming.